dashboard/app/models/concerns/serialized_properties.rb

```diff
@@ -10,7 +10,7 @@ module SerializedProperties
     self.serialized_properties ||= {}
 
     after_initialize :init_properties
-    before_save {properties.select! {|_, v| v.present?}}
+    before_save {properties.select! {|_, v| !v.nil?}}
```

Comment:
This might have some unintended consequences for how we store data, since `present?` removes things like empty objects and arrays. A good first check will be if this causes test failures on drone. It might also be worth spinning up a thread on #dashboard-dev as to what the best approach here might be--I can see the benefits of including 'false' values but not empty objects, for example. If it causes a lot of issues it might be worth considering changing usingDarkMode from a true/false to instead be a string variable 'displayMode' that is either 'light' or 'dark'.

Reply:
Yep, makes sense - I'll wait and see if the drone build passes, but agree that this certainly could have some unintended consequences. I think your suggestion of changing the field to a string makes sense, especially with Jessie's comment below.

Reply:
Looks like this did break some tests in the drone build; I think I'll discard this change and instead retool the solution to use an string/enum solution as suggested instead so we don't need to touch this code.